Add 'activities' endpoint support

jjlawren opened the pull request:
PMS reports back information on its currently running activities, such as scanning for metadata.

This is an MVP version which should allow querying to see if metadata activities are still in progress as suggested [here](https://github.com/pkkid/python-plexapi/issues/454#issuecomment-691329452).

maykar commented on plexapi/server.py:
```diff
@@ -190,6 +190,14 @@ def account(self):
+        activities = []
+        for elem in self.query(Activity.key):
+            activities.append(Activity(self, elem))
+        return activities
```
```py
    @property
    def activities(self):
        """Returns all current PMS activities."""
        return [Activity(self, elem) for elem in self.query(Activity.key)]
```

jjlawren commented:
I've added a test that's intended to avoid some test failures based on incomplete metadata. The tests that rely on metadata agents will now wait to run until all server activities have finished.

I ran a "canary" test in this PR which showed that agents are sometimes still scanning when we begin validating metadata correctness in the CI test runs.

jjlawren commented:
Moved it to the start of the run (alphabetically) and added an authenticated version so it's guaranteed to run first in both auth and anon runs.

I'm still not sure how this is failing since there's an explicit check in [`plex-bootstraptest.py`](https://github.com/pkkid/python-plexapi/blob/fb82bc402bd725220687b76c01a35008884b75ac/tools/plex-bootstraptest.py#L279-L280) that tries to wait for metadata scans. In the canary test that failed the bootstrap script seemed to believe the scanning was complete. Maybe we need multiple different methods to be sure.

jjlawren commented:
Well, that theory didn't pan out. After multiple successful runs, the latest CI run passed the activity check but still failed in the audio metadata:
https://travis-ci.org/github/pkkid/python-plexapi/jobs/727760606#L839-L873

jjlawren commented:
This didn't prevent the metadata test failures, but I don't see a downside to including these tests as a failsafe. Will merge alongside #571 and see how the CI behaves before making more changes to the tests.
